### ISSUE TITLE
Process images in parallel

### ISSUE THREAD

**Issue by schwartzadev** [COVID19Tracking/contentful-image-upload#2]
Using an async library, let's upload the images to contentful in parallel.

This looks like:
1. Creating a thread for each image
2. Within each thread, create:
  * an `upload`,
  * an `asset`,
  * and an Image Content Block `entry`

(low priority: let's also optimize the images in parallel.)
